Clapet/ClapetApp.swift

```diff
@@ -55,11 +55,11 @@ struct ClapetApp: App {
         }
         
         Settings {
-            if alreadySetup {
+//            if alreadySetup {
```

Comment:
`alreadySetup` is false on the first launch and gets toggled to true on the introduction screen when the user either clicks on skip or the configure automatically button, has it caused any problem during development or use ?

<img width="795" alt="image" src="https://github.com/user-attachments/assets/4eeb7923-9a5c-4a8e-8060-2650765bb0af">

Reply:
For some reason, this window never shows up on MacOS 15. Perhaps I could retry but not entirely sure whats the best way to fully "reset" the app

Reply:
There's a reset button in the advanced settings, it resets all the @AppStorage variables back to their default values, however it also automatically sets alreadySetup back to true, so you can comment line 54 in AdvancedSettings.swift and click on the reset settings button, then restart the app and the introduction screen should open

<img width="477" alt="image" src="https://github.com/user-attachments/assets/65b88443-94a3-4012-9c6d-142c0b560066">